index.html

```diff
@@ -4,9 +4,37 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/style.css">
+  <script src="https://kit.fontawesome.com/83eb6c852a.js" crossorigin="anonymous"></script>
+
   <title>My Portfolio: Emmanuel Orji-Ihuoma</title>
 </head>
 <body>
+
+  <header>
+    <nav class="toolbar">
+      <a href="/"><h1>Emmanuel Orji</h1></a>
+      <i class="fa-solid fa-bars"></i>
+    </nav>
+    <section class="headline-container">
+      <div class="headline">
+        <h1>
+          <span>Hey There.</span>
+          <span>I'm Emma</span>
+        </h1>
+        <h2>I am a Software Developer</h2>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</p>
+        <div class="socials">
+          <a href="https://github.com/emmiiorji"><i class="fa-brands fa-github"></i></a>
+          <a href="https://www.linkedin.com/in/emmanuel-orji-2a8317121"><i class="fa-brands fa-linkedin-in"></i></a>
+          <a href="#"><i class="fa-brands fa-angellist"></i></a>
+          <a href="#"><i class="fa-brands fa-twitter"></i></a>
+          <a href="#"><i class="fa-brands fa-medium"></i></a>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Please remove the `section tag` out of the `header tag`. The `section` should be on its own, it should be the next part after the `header tag`.